m4_sessions/1801-inning/group_5.md

```diff
@@ -22,8 +22,29 @@ Students should leave with more confidence and some useful skills/strategies on
 
 Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
 
-15 minutes intros
-15 minutes how could your intro have been more impactful? asking questions, being genuinely interested in others
-15 minutes identifying weaknesses, what holds us back from speaking out more?
-15 minutes strength of loose ties - maintaining loose network connections, weakness of strong ties - how are your close relationships?
-15 minutes wrap up, practice, discussion 
+5 minutes (We all introduce ourselves by name and Mod)
+ - We will split the group into pairs. The pairs will take turns introducing themselves to each other.
+ 
+ -- Catch everyone off guard by asking them to introduce their partners to the group --
+ 
+ 5 minutes (We will be walking around answering questions)
+ - Everyone has to think for themselves on how they'd want to be introduced.
+   - Questions for preparing a bio... 
+    - What are you doing currently? What were you doing before? Also... something unique/exciting/valuable?
+ 
+10 minutes
+  Then each person will introduce themselves to their partner again
+
+10-15 minutes to introduce their partners to the group
+
+15 minutes (Coordinated by all of us)
```

Comment:
Would be great to add a few resources here--articles on body language, how to use it to your benefit